server/controllers/entities/lib/properties/properties_values_constraints.coffee

```diff
@@ -123,6 +133,8 @@ module.exports =
   'wdt:P2963': builders.externalId /^[1-9]\d*$/
   # GoodReads book ID
   'wdt:P2969': builders.externalId /^[1-9]\d*$/
+  # ISBN publisher prefix
+  'wdt:P3035': builders.externalId /^978-\d-\d{1,}$/
```

Comment:
allow `979` prefixes and be more restrictive on the following blocks (copying [P3035#P1793](https://www.wikidata.org/wiki/Property:P3035#P1793) regex)
```suggestion
  'wdt:P3035': builders.externalId /^97(8|9)-\d{1,5}-\d{2,7}$/
```